images/govready-q/wait-for-database.py

```diff
@@ -0,0 +1,17 @@
+import os
+import re
+import socket
+import time
+
+host, port = os.environ['DATABASE_CONNECTION_STRING'].split('@')[-1].split('/')[0].split(':')
+
+s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+while True:
```

Comment:
IMO it'd be beneficial to add a timeout option here

Reply:
The timeout is being handled on the healthcheck for the `app` service on the compose file.  This python script can run forever and will eventually be terminated when the stack comes down because it exceeded the healthcheck for this service.  For that reason there is no need to put an extra timeout in place at this time.